public/js/torrents/index.js

```diff
@@ -0,0 +1,3 @@
+$(function(){
```

Comment:
This is halfway there:

On how to integrate this like Dennis suggested, pls check: https://github.com/wdi-hk-9/WDI-Express/compare/frontend-api

![image](https://cloud.githubusercontent.com/assets/970858/11390455/369535f4-9385-11e5-9cd9-b7d111c55b5b.png)